src/logic/install.ts

```diff
@@ -204,28 +204,26 @@ export async function resolveInstallationRequests(
     }
   }
 
+  await generateTypes(phase3, superJson);
```

Comment:
Can I has the doc on this function updated? With like `5. generate typings` or something

Reply:
what value will that add?

Reply:
Inner peace and not having to read the code when you just want to see where the typings are generated

Reply:
I mainly meant the enclosing function (line 140) 🙏